src/operator-docs/transformation/partition.ts

```diff
@@ -1,6 +1,69 @@
 import { OperatorDoc } from '../operator.model';
 
 export const partition: OperatorDoc = {
-  'name': 'partition',
-  'operatorType': 'transformation'
+  name: 'partition',
+  operatorType: 'transformation',
+  signature:
+    'public partition(predicate: function(value: T, index: number): boolean, thisArg: any): [Observable<T>, Observable<T>',
```

Comment:
here is a little syntax error. At the end, there is a missing ]

Reply:
hmm i see...